src/land/smap_ssm2ioda.py

```diff
@@ -90,10 +90,8 @@ def _read(self):
         lons = ncd.groups['Soil_Moisture_Retrieval_Data'].variables['longitude'][:].ravel()
         errs = ncd.groups['Soil_Moisture_Retrieval_Data'].variables['soil_moisture_error'][:].ravel()
         qflg = ncd.groups['Soil_Moisture_Retrieval_Data'].variables['retrieval_qual_flag'][:].ravel()
-
         deps = np.full_like(vals, self.assumedSoilDepth)
-        times = np.empty_like(vals, dtype=np.int64)
-
+        times = np.empty_like(vals, dtype=object)
```

Comment:
@YoulongXia-NOAA and @zcstanley didn't see any conversion of this string time (from the filename) to integer seconds so leaving as is for future update

Reply:
@BenjaminRuston, see this part:
# get datetime from filename
        str_split = self.filename.split("_")
        str_datetime = str_split[7]
        my_datetime = datetime.strptime(str_datetime, "%Y%m%dT%H%M%S")
        time_offset = round((my_datetime - epoch).total_seconds())